doc/level1.rst

```diff
@@ -275,6 +275,23 @@ above rules, such headers will be judged invalid, and discarded by
 level 1 MUAs.  Such an update to the specification will also have
 to describe how an updated MUA will deal with multiple valid headers.
 
+A MUA MUST NOT send Autocrypt headers above 10 KiB in size
+(including the Autocrypt: prefix and folding whitespace).
+Conversely, MUAs SHOULD not process Autocrypt headers
```

Comment:
Again, i would prefer singular here.  and what should the MUA do if it finds an Autocrypt header that is > 10KiB in size?  should it treat the header as non-existent?  as an invalid or mal-formed Autocrypt header?  Those might be subtly different, and we should give clear guidance.

Reply:
works for me :)